src/Eveonline/composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
-        "firebase/php-jwt": "^5.2",
+        "firebase/php-jwt": "^6.0",
```

Comment:
@atymic why not ^5.2 || ^6.0 ?

Reply:
cc @billypoke 
cf. https://github.com/SocialiteProviders/Providers/pull/803

Reply:
See #807